<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::SensorView</class>
 <widget class="QWidget" name="QGBA::SensorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sensors</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Realtime clock</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QRadioButton" name="timeNoOverride">
          <property name="text">
           <string>System time</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">timeButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="timeFixed">
          <property name="text">
           <string>Fixed time</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">timeButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="timeNow">
          <property name="text">
           <string>Now</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="timeOffset">
          <property name="text">
           <string>Offset time</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">timeButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="offsetSeconds">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> sec</string>
          </property>
          <property name="minimum">
           <number>-99999999</number>
          </property>
          <property name="maximum">
           <number>99999999</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="timeFakeEpoch">
          <property name="text">
           <string>Start time at</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">timeButtons</string>
          </attribute>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QDateTimeEdit" name="time">
          <property name="wrapping">
           <bool>true</bool>
          </property>
          <property name="maximumDate">
           <date>
            <year>2099</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="currentSection">
           <enum>QDateTimeEdit::MonthSection</enum>
          </property>
          <property name="displayFormat">
           <string>MM/dd/yy hh:mm:ss AP</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Light sensor</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Brightness</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="lightSpin">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QSlider" name="lightSlide">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="tilt">
       <property name="title">
        <string>Tilt sensor</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1">
           <widget class="QPushButton" name="tiltSetY">
            <property name="text">
             <string>Set Y</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="tiltSetX">
            <property name="text">
             <string>Set X</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" alignment="Qt::AlignHCenter">
           <widget class="QDoubleSpinBox" name="tiltX">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1" alignment="Qt::AlignHCenter">
           <widget class="QDoubleSpinBox" name="tiltY">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gyro">
       <property name="title">
        <string>Gyroscope</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="0" column="1">
           <widget class="QPushButton" name="gyroSetY">
            <property name="text">
             <string>Set Y</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="gyroSetX">
            <property name="text">
             <string>Set X</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Sensitivity</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="gyroSensitivity">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QSlider" name="gyroView">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-1073741823</number>
            </property>
            <property name="maximum">
             <number>1073741823</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="timeButtons"/>
 </buttongroups>
</ui>
